pillarbox-demo/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2022. SRG SSR. All rights reserved.
+  ~ License information is available from the LICENSE file.
+  -->
+<resources>
+    <item name="streams" type="id" />
+    <integer name="info" type="id" />
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SRGSSR/pillarbox-android/blob/9e672e88664d24e42b2fbb65a69ff4507dff46b6/pillarbox-demo/src/main/res/values/ids.xml#L7"></span>The resource <code>R.integer.info</code> appears to be unused</td>
    </tr>
  </tbody>
</table>